controller/logs.go

```diff
@@ -0,0 +1,60 @@
+package controller
+
+import (
+	"context"
+	"fmt"
+	"math"
+	"strings"
+	"time"
+)
+
+func (c *Controller) GetActiveDeploymentLogs(ctx context.Context, numLines int32) error {
+	projectID, err := c.cfg.GetProject()
+	if err != nil {
+		return err
+	}
+	environmentID, err := c.cfg.GetEnvironment()
+	if err != nil {
+		return err
+	}
+	deployments, err := c.gtwy.GetDeploymentsForEnvironment(ctx, projectID, environmentID)
+	if err != nil {
+		return err
+	}
+
+	latestDeploy := deployments[0]
```

Comment:
Should we be checking `len(deployments)` here?

Might also be useful to have a `GetLatestDeploymenForEnvironment` fn later on with this logic contained.

Reply:
I can slap that into the gateway

Reply:
FWIW that query might be useful to have in Backboard. Also might want to check if the latest deploy status is not REMOVED.

Reply:
I think it's fine to live in the CLI for now. If we see more usecases for it, we can pull it upwards

Will also filter out removed deployments good call